### PR TITLE
Adding missing apps to the static app descriptor files

### LINKED ISSUE #229: Verify all apps are included in the static app descriptor

### PR DESCRIPTION
**Pull request by sobychacko**
Resolves #229

### CODE REVIEW

File: spring-cloud-stream-app-descriptor/src/main/resources/META-INF/kafka-apps-docker.properties [hunk 1]
```diff
@@ -43,6 +45,9 @@ sink.rabbit=docker:springcloudstream/rabbit-sink-kafka:@docker.tag@
 sink.redis-pubsub=docker:springcloudstream/redis-sink-kafka:@docker.tag@
 sink.router=docker:springcloudstream/router-sink-kafka:@docker.tag@
 sink.s3=docker:springcloudstream/s3-sink-kafka:@docker.tag@
+sink.sftp=docker:springcloudstream/sftp-sink-kafka:@docker.tag@
+sink.task-launcher-local=docker:springcloudstream/task-launcher-local-sink-kafka:@docker.tag@
+sink.task-launcher-yarn=docker:springcloudstream/task-launcher-yarn-sink-kafka:@docker.tag@
```
Comment (ilayaperumalg) [inline spring-cloud-stream-app-descriptor/src/main/resources/META-INF/kafka-apps-docker.properties:50]: Looks good. Do you think we need to include the processor `integration-test` as it is included as part of the `apps`?

Comment (ericbottard) [reply spring-cloud-stream-app-descriptor/src/main/resources/META-INF/kafka-apps-docker.properties:50]: `integration-test` is only used by the automated tests, there is no reason to register it in SCDF

Comment (sobychacko) [inline spring-cloud-stream-app-descriptor/src/main/resources/META-INF/kafka-apps-docker.properties]: I think we can add that.

**Comment by ilayaperumalg** [comment]
Rebased and merged as 5a69bd1

File: spring-cloud-stream-app-descriptor/src/main/resources/META-INF/kafka-apps-docker.properties (hunk 1 above)
Comment (ilayaperumalg) [reply spring-cloud-stream-app-descriptor/src/main/resources/META-INF/kafka-apps-docker.properties:50]: ok, thanks for the confirmation @ericbottard

**Pull request closed without merging**